meta-refkit-core/conf/distro/include/enable-refkit-config.inc

```diff
@@ -15,3 +15,6 @@ VIRTUAL-RUNTIME_initscripts = ""
 # defaults from OE-core.
 DISTRO_FEATURES_DEFAULT_remove = "${REFKIT_DEFAULT_DISTRO_FEATURES_REMOVE}"
 DISTRO_FEATURES ?= "${DISTRO_FEATURES_DEFAULT} ${DISTRO_FEATURES_LIBC} ${REFKIT_DEFAULT_DISTRO_FEATURES}"
+
+require conf/distro/include/usrmerge.inc
+require conf/distro/include/flatpak.inc
```

Comment:
Interesting. All the tests passed in my test build without any such failures (at least I did not stop any). I don't want to interfere now/any more with Mikkos release attempts if he's willing to take this as such, but I change these to includes and do queue them for my next PR.